pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -479,9 +479,10 @@ func (o *SyncOptions) enableSyncerForWorkspace(ctx context.Context, config *rest
 			Resources:     []string{"synctargets"},
 		},
 		{
-			Verbs:     []string{"get", "list", "watch"},
-			APIGroups: []string{workloadv1alpha1.SchemeGroupVersion.Group},
-			Resources: []string{"synctargets"},
+			Verbs:         []string{"get", "list", "watch"},
+			APIGroups:     []string{workloadv1alpha1.SchemeGroupVersion.Group},
+			Resources:     []string{"synctargets"},
+			ResourceNames: []string{syncTargetName},
```

Comment:
just curious, maybe not necessary to implement:

If we needed this in a production system, i.e. reduce permissions and make people update the syncer. How would we do that?

Reply:
seems to me this permission should be controlled by controller instead of users?